### ISSUE TITLE
ogbn-products submission

### ISSUE THREAD

**Issue by weihua916** [GraphSAINT/GraphSAINT#19]
Hi, thanks for your leaderboard submission to OGB.
I could not find something like `from ogb.nodeproppred import PygNodePropPredDataset`. How did you obtain the dataset? Also, please put add `README.md` at https://github.com/GraphSAINT/GraphSAINT/tree/master/graphsaint/pytorch_version to give details on how to reproduce your results.

Edit: I found the dataset importing at `ogb_converter.py`. In `README.md`, could you describe the step-by-step process of how you can obtain the results? 

Thanks!!

**Comment by ZimpleX** [comment]
Sorry about the confusion! We just cleaned up a bit to clarify the procedure of reproducing the results. 

Yes, the data conversion is by `ogb_converter.py` (now renamed to `ogbn_converter.py`).

The training script is now moved under `./graphsaint/open_graph_benchmark/`. The command to reproduce accuracy is given in `./graphsaint/open_graph_benchmark/README.md`.

Please let us know if there is further concerns. 

Thanks!!

**Comment by weihua916** [comment]
Perfect, thanks!! I have just updated the [leaderboard](https://ogb.stanford.edu/docs/leader_nodeprop/#ogbn-products).
